src/state/wallets/selectors.ts

```diff
@@ -99,6 +104,17 @@ export const allWallets = createSelector(wallets, allWallet, (walletsList, aw) =
   return walletsList;
 });
 
+export const subscribedWallets = createSelector(store, wallets, (state, walletsList) =>
+  walletsList.filter(wallet => state.notifications.subscribedIds.some(id => id === wallet.id)),
+);
+
+export const unSubscribedWallets = createSelector(store, wallets, (state, walletsList) =>
+  walletsList.filter(
+    wallet =>
+      !state.notifications.subscribedIds.some(id => id === wallet.id) && wallet.type === HDSegwitP2SHAirWallet.type, //till all wallets are possible to subscribe
```

Comment:
you added selector for subscribedIds
from what I know, all wallets should be able to sub/unsub

Reply:
If the backend is not ready for it yet, the app will get crash.

Reply:
anyway, selector subscribedIds from notifications selectors should be used instead of state.notifications.subscribedIds